numpy/add_newdocs.py

```diff
@@ -7756,3 +7758,36 @@ def luf(lamdaexpr, *args, **kwargs):
 
 add_newdoc('numpy.core.numerictypes', 'object_',
     """Any Python object.  Character code: 'O'.""")
+
```

Comment:
What does add_newdoc actually do in this case?  It seems like usually it ends up in __umath_generated.c, but not in this case since there isn't anything in generate_umath.py